### PR TITLE
Add installation info on Mozart 2 Homebrew cask to MacOS README

### PR DESCRIPTION
**Pull request by dskecse**
Installing Mozart 2 as a Homebrew cask could be much easier and less time consuming for end users than building it, especially if they don't have plans to work on it, so they could be set up in virtually no time.

/cc @ggutierrez @eregon

### CODE REVIEW

**Comment by eregon** [comment]
So this is a convenience over the sourceforge download essentially?

File: README.MacOS.md [hunk 1]
```diff
@@ -10,7 +10,20 @@ Download and install the following tools:
 *  [Homebrew](http://mxcl.github.com/homebrew/).
 *   Any emacs distribution, for instance [Aquamacs](http://aquamacs.org/).
 
-### Homebrew tap
+### Homebrew cask
+
+In case you don't have plans to work on Mozart 2, you can simply install it as
+a compiled binary via [Homebrew cask] (if you have Homebrew installed you're
+already set up):
```
Comment (eregon) [inline README.MacOS.md]: This text disappears in the rendered version. Maybe just move it to the next line.

Comment (sjmackenzie) [inline README.MacOS.md]: Reverted

Comment (eregon) [inline README.MacOS.md]: Fixed in 7147ed72ec74b790ea0a5536c1a4aba7488403f1

**Pull request merged**